src/Symfony/Component/Validator/Constraints/NullValidator.php

```diff
@@ -18,6 +18,8 @@
  * @author Bernhard Schussek <bschussek@gmail.com>
  *
  * @api
+ *
+ * @deprecated
```

Comment:
`@deprecated since version 2.3.28, to be removed in 3.0. Use IsNullValidator instead.`